scylla/src/client/session.rs

```diff
@@ -347,24 +347,24 @@ pub(crate) enum RunRequestResult<ResT> {
 
 impl Session {
     /// Sends a request to the database and receives a response.\
-    /// Performs an unpaged query, i.e. all results are received in a single response.
+    /// Executes an unprepared CQL statement without paging, i.e. all results are received in a single response.
```

Comment:
:thought_balloon: :+1: This makes sense to me. We _send a CQL (protocol) request_ and, this way, we _execute a CQL (language) statement_, or, more precisely, we _have the CQL statement executed_ on the server.